docs/development/releasing.md

```diff
@@ -2,21 +2,32 @@
 
 ## Release Branch
 
-Always create a release branch for the releases, for example branch `release-0.5` is for all the v0.5.x versions release. If it's a new release branch, simply create a branch from `main`.
+Always create a release branch for the releases. For example branch `release-0.5` is for all the `v0.5.x` version releases. If it's a new release branch, simply create a branch from `main`.
```

Comment:
`releasing.md` is supposed to be read by contributors/developers, we could mention something like "version mapping needs to be updated at xxx when needed". The real version mapping would be a doc for the users, instead of using a yaml file, it would be better to be in a doc file, maybe shown as a table.